packages/css/src/viewTransition.ts

```diff
@@ -0,0 +1,13 @@
+import { appendCss } from './adapter';
+import { getFileScope } from './fileScope';
+import { generateIdentifier } from './identifier';
+import { CSSViewTransition } from './types';
+
+// createViewTransition is used for locally scoping CSS view transitions
+// For now it is mostly just an alias of generateIdentifier
+export const createViewTransition = (debugId?: string) =>
+  generateIdentifier(debugId);
+
+export function viewTransition(rule: CSSViewTransition) {
+  appendCss({ type: 'view-transition', rule }, getFileScope());
+}
```

Comment:
There's a bit more to adding a `@view-transition` API than just adding this function. The actual generation of CSS needs to be implemented to. Happy to point you in the right direction if you want to implement it, but I was planning to do it in a PR separate to this one.

If you need to use VE to write a view transition rule, you can currently just use a global style as a hack:

```ts
globalStyle('@view-transition': {
  navigation: 'auto',
})
```

Otherwise, the `createViewTransition` stuff looks ready to release though.

Reply:
I can remove it to get this out! Just was playing with getting it added, but makes sense to just get this merged.